packages/analytics/src/rudderstack.ts

```diff
@@ -133,14 +134,68 @@ type IdentifyAction = {
     language: string;
 };
 
+type ReportsFormAction =
```

Comment:
Should we make use of the constant values that you've defined in `rudderstack-events.ts`?

Reply:
It's a type, I don't think so. If the type just checks whether the variable has the same value assigned as what was assigned to it, then what's the point of having typescript at all?